components/common/src/templating/config.rs

```diff
@@ -613,10 +613,12 @@ mod test {
 
     fn curr_username() -> String {
         users::get_current_username().expect("Can get current username")
+                                     .expect("Can get current username")
```

Comment:
I know this is a test helper function, but it would be good to make these `expect` messages different to help distinguish which operation failed, if one of them ever does.

I see a handful of other instances of this pattern elsewhere; same suggestion applies.

Reply:
Agreed. Addressed in https://github.com/habitat-sh/habitat/pull/7879/commits/1bd6407da4aecc1299a5ac0eef5ddcb0a9dbc46c